jupyterlab/README.md

```diff
@@ -1,14 +1,33 @@
-### Docker build
+### `pip` based version
+
+#### Docker build
+
+```aidl
+docker build -t activeeon/jupyterlab:pip -f ./Dockerfile.pip .
 ```
-docker build -t activeeon/jupyterlab .
+
+#### Docker run
+
+```aidl
+docker run -it -p 8888:8888 activeeon/jupyterlab:pip
 ```
 
-### Docker run
+### `conda` based version
+
+#### Docker build
+
+```aidl
+docker build -t activeeon/jupyterlab:conda -f ./Dockerfile.conda .
 ```
-docker run -it -p 8888:8888 activeeon/jupyterlab
+
+#### Docker run
+
+```aidl
+docker run -it -p 8888:8888 activeeon/jupyterlab:conda
```

Comment:
do you really need interactive mode? detached maybe?

Reply:
@mboussaa yes...this docker container is intended to be used as a open service.